### ISSUE TITLE
Failure in inverse folding downstream task

### ISSUE THREAD

**Issue by Lim-ZQ** [genbio-ai/ModelGenerator#23]
I was trying to reproducing benchmark result of inverse foldng task. I can successfully reproduce gRNAde+AIDO.RNA-cDiffusion result which is 54.41. However, I cannot successfully reproduce gRNAde+AIDO.RNA-zeroshot result which is 53.16.

I did zeroshot prediction by not setting the finetuned checkpoint which will use the base model by default. 

```
mgen test --config ./experiments/AIDO.RNA/rna_inverse_folding/rna_inv_fold_test.yaml \
 --trainer.default_root_dir ${MGEN_DATA_DIR}/modelgenerator/logs/rna_inv_fold/ \
##### --ckpt_path ${MGEN_DATA_DIR}/modelgenerator/huggingface_models/rna_inv_fold/AIDO.RNAIF-1.6B/model.ckpt \
 --trainer.devices 0, \
 --data.path ${MGEN_DATA_DIR}/modelgenerator/datasets/rna_inv_fold/structure_encoding/\
```

However by this way the prediction will still have [MASK] token which causes error. I need to add a new letter 'M' when decoding output

```
        bases = ["A", "G", "C", "U", "M"]
        if self.trainer.testing:
            self.true_sequences += [
                "".join(
                    [bases[x] for x in target_ids[0].detach().cpu().numpy().tolist()]
                )
            ]
            
            print(pred_tokens[0].detach().cpu().numpy().tolist())
            self.pred_sequences += [
                "".join(
                    [bases[x] for x in pred_tokens[0].detach().cpu().numpy().tolist()]
                )
            ]
            # print(pred_tokens[0].detach().cpu().numpy().tolist())
            self.baseline_sequences += [
                "".join(
                    [
                        bases[x]
                        for x in data["str_enc_logits"]
                        .argmax(-1)[0]
                        .detach()
                        .cpu()
                        .numpy()
                        .tolist()
                    ]
                )
            ]
```

What's more, the accuracy is only 51 due to several [MASK] token in prediction.
┏━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃        Test metric        ┃       DataLoader 0        ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
│         test/acc          │    0.5108641982078552     │
│         test/loss         │    1.6362208127975464     │
│         test_acc          │    0.5109999775886536     │
└───────────────────────────┴───────────────────────────┘

Is this the correct way to reproduce zeroshot prediction? As the base model are not in a single file like 'model.ckpt', I cannot load it like a fintuned model

**Comment by cnellington** [comment]
Hi @Lim-ZQ, thanks for your patience with this, the updates got caught in the middle of a major release. Now that it's out, you can find comprehensive docs for reproducing this experiment from @Sazan-Mahbub here:
https://github.com/genbio-ai/ModelGenerator/tree/main/experiments/AIDO.RNA/rna_inverse_folding
